app/controllers/conference_attendances_controller.rb

```diff
@@ -11,6 +11,7 @@ def update
   end
 
   private
+
     def attendance_params
```

Comment:
Shouldn't this extra level of indentation be caught by RC?

Reply:
Yes, but it's a different cop we don't yet have. But if we already have the one for a blank line before access modifiers and its indentation, it just makes sense to also add the one for proper (in my opinion 😉 ) indentation after it as well.

I'll add it ✌️